erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -221,7 +221,7 @@ def set_per_picked(self):
 		"""
 		all_orders = [d.sales_order for d in self.locations]
 		for order in unique(all_orders):
-			picked_qty = sum(d.picked_qty for d in self.locations if d.sales_order == order)
+			picked_qty = sum(d.picked_qty for d in self.locations if d.sales_order == order and d.picked_qty)
 			ordered_qty = sum(d.qty for d in self.locations if d.sales_order == order)
```

Comment:
@MyuddinKhatri if order_qty is not mandatory then do the same changes for ordered_qty